### PR TITLE
add my image

### PR DESCRIPTION
**Pull request by kol4k**
gci

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -288,7 +288,7 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
     <div class="row">
     <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
-		                    <img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
+		                    <img class="col-sm-2" alt="Mochammad Rafi Al Rizkan" src="img/rafi.png">
```
Comment (abishekvashok) [inline index.html]: Don't change an existing student's image

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by kol4k** [comment]
`
		                <div class="col-sm-12 text-center spread-children-large">
							Miguel Piedrafita
		                    Chahat Bansal
		                    Nitesh Kumar Niranjan
		                    Aleksandr Lymarenko
		                    Naveen Rajan
		                    Abishek V Ashok
		                    Mochammad Rafi Al Rizkan
		                </div>`
i've add again,

File: index.html (hunk 1 above)
Comment (kol4k) [reply index.html]: how to i pull request again?
```
		                <div class="col-sm-12 text-center spread-children-large">
					<img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
		                    <img class="col-sm-2" alt="Chahat Bansal" src="img/chahat.png">
		                    <img class="col-sm-2" alt="Nitesh Kumar Niranjan" src="img/niteshkumarniranjan.jpg">
		                    <img class="col-sm-2" alt="Aleksandr Lymarenko" src="img/a-v-lymarenko.png">
		                    <img class="col-sm-2" alt="Naveen Rajan" src="img/naveen.jpg">
		                    <img class="col-sm-2" alt="Abishek V Ashok" src="img/abishek.jpg">
		                    <img class="col-sm-2" alt="Mochammad Rafi Al Rizkan" src="img/rafi.png">
		                </div>
```

**Comment by kol4k** [comment]
ive been adding new commit.

**Comment by kol4k** [comment]
i can't add new pull

**Comment by abishekvashok** [comment]
@kol4k that's ok, now coan you squash them?

File: index.html [hunk 2]
```diff
@@ -288,12 +288,13 @@ <h4 class="uppercase mb16">Lightbox Grid Gallery</h4>
     <div class="row">
     <div class="row mb64 mb-xs-24">
 		                <div class="col-sm-12 text-center spread-children-large">
-		                    <img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
+							<img class="col-sm-2" alt="Miguel Piedrafita" src="img/m1guelpf.png">
```
Comment (abishekvashok) [inline index.html:291]: Let's keep it professional, don't deintedn the code, i meant the blank lines.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (kol4k) [reply index.html:291]: no. the line no blank.

Comment (abishekvashok) [reply index.html:291]: See the giff. You are adding some spaces before the  tag :)

**Review by jigyasa-grover — CHANGES_REQUESTED** [review]
Please provide a live link for testing, thanks.

**Comment by kol4k** [comment]
https://kol4k.github.io/gci17.fossasia.org/
In my repository is update and fix it.

**Comment by isuruAb** [comment]
@kol4k  nice works. could you please squash these two commits into one ? cheers 👍

**Comment by kol4k** [comment]
ok. i've been added two commits into one :D

**Comment by isuruAb** [comment]
@kol4k watch this tutorial for squash commits.
https://www.youtube.com/watch?v=vp7-NeURmFg&feature=youtu.be

**Comment by kol4k** [comment]
yes, i've been added squash commits. please recheck my pull code https://github.com/fossasia/gci17.fossasia.org/pull/60/commits/18d8d7be9bd658a47131272e26a00c6d99481c3e

**Review by isuruAb — APPROVED** [review]

**Comment by harshit98** [comment]
@kol4k ! Good job!
I'm merging this pull request. Hope to see more contributions from your side. :)

**Review by harshit98 — APPROVED** [review]

**Pull request merged**